### PR TITLE
start of parser table

### PR DESCRIPTION
**Pull request by rjpepmeyer**
Concerns issue #6

### CODE REVIEW

File: parser.cpp [hunk 1]
```diff
@@ -1,5 +1,86 @@
...
+      else if (col.type == boolean) return 15;
+      else return 0;
+    case expr:
+      if (col.value == "if" || col.value == "not" || 
+      col.type == identifier || col.type == number ||
+      col.type == boolean || col.value == "-" || 
+      col.value = "(") return 17;
+      else return 0;
+    case exprprime:
+      //TODO
```
Comment (dmnemec) [inline parser.cpp]: `if (col.value == "<") return 1;`
`else if (col.value == "=") return 2;`

File: parser.cpp [hunk 2]
```diff
@@ -1,5 +1,86 @@
...
+      else return 0;
+    case simpleexpr:
+      if (col.value == "if" || col.value == "not" || 
+      col.type == identifier || col.type == number ||
+      col.type == boolean || col.value == "-" || 
+      col.value = "(") return 21;
```
Comment (DMNemec) [inline parser.cpp]: should be "return 2" according to the parsing table

File: parser.cpp (hunk 1 above)
Comment (dmnemec) [reply parser.cpp]: not sure if this is correct yet, as this may not have been entered properly in the parsing table.

Comment (rjpepmeyer) [reply parser.cpp]: I filled out the rest so it's ready for bug fixing now

**Pull request merged**